src/main/kotlin/org/rust/lang/core/psi/impl/mixin/RustItemImplMixin.kt

```diff
@@ -26,15 +27,28 @@ abstract class RustItemImplMixin : RustStubbedNamedElementImpl<RustItemStub>
 
         override fun getPresentableText(): String? = name
     }
+
+    override val documentation: String?
```

Comment:
So, let's implement this as an extension method on `RustItem`:

```
val RustItem.documentation: String? get {
    ...
}
```

You can place it in the `RustItemImplMixin.kt` file